demosymfonyform/src/Controller/DemoConfigurationController.php

```diff
@@ -40,7 +40,7 @@ public function index(Request $request): Response
         $form = $formDataHander->getForm();
         $form->handleRequest($request);
 
-        if ($form->isSubmitted()) {
+        if ($form->isSubmitted() && $form->isValid()) {
```

Comment:
Is it relevant to save the form if it is not valid? (true question)

I guess you will have both Form constraint displayed inline and Domain exception errors displayed in the notifications. But you can also endup with doublons between notif and inline errors.

And the worst would be that server side is not as accurate as the Form in its constraint and returns no errors, then the form will be indeed saved (with invalid inputs) and no errors will be displayed.

Reply:
>Is it relevant to save the form if it is not valid? (true question)

You shouldn't save an invalid form.  This is why the check was missing :thinking: